app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -180,8 +177,9 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
     }
 
     @Override
-    public void onResume() {
-        super.onResume();
+    public void onStart() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'onStart()' has an NPath complexity of 500](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5359905)